docker/launch-hostagent.sh

```diff
@@ -156,6 +156,11 @@ if [ -w ${PREFIX} ]; then
     else
         mkdir -p "${VARDIR}/lib/opflex-agent-ovs/netpols"
     fi
+    if [ -d "${VARDIR}/lib/opflex-agent-ovs/OutOfBandConfig" ]; then
+        rm -rf "${VARDIR}/lib/opflex-agent-ovs/OutOfBandConfig"/*
```

Comment:
In keeping with the rest of the directories there, lets call this directory just /lib/opflex-agent-ovs/outofband. Filename  is fine.